test/FractionalPool.t.sol

```diff
@@ -115,7 +116,7 @@ contract Deployment is FractionalPoolTest {
 
 contract Deposit is FractionalPoolTest {
   function test_UserCanDepositGovTokens(address _holder, uint256 _amount) public {
-    _amount = bound(_amount, 0, type(uint224).max);
+    _amount = bound(_amount, 0, type(uint208).max);
```

Comment:
Why this was changed. https://github.com/OpenZeppelin/openzeppelin-contracts/blob/920225a1c7e6e749ba23b10ed076b04f454290e8/contracts/token/ERC20/extensions/ERC20Votes.sol#L40

Reply:
Because `Votes.sol` uses `Checkpoints.Trace208` for the checkpoints. This is a consequence of the adoption of `uint48` for all clocks/timestamps, following ERC-5805